ldms/src/sampler/lustre_mdt/lustre_mdt.c

```diff
@@ -84,7 +89,7 @@ static struct mdt_data *mdt_create(const char *mdt_name, const char *basedir)
         mdt->general_metric_set = mdt_general_create(producer_name, mdt->fs_name, mdt->name, &cid);
         if (mdt->general_metric_set == NULL)
                 goto out7;
-        mdt->osd_path = mdt_general_osd_path_find(OSD_SEARCH_PATH, mdt->name);
+        mdt->osd_path = mdt_general_osd_path_find(possible_osd_base_paths, mdt->name);
```

Comment:
This is does not work. mdt_general_osd_path_find() tasks a single string as its first parameter. We can't just change that to being a list without also changing the code of mdt_general_osd_path_find().

Reply:
I have no idea what is going on. At the time of my review, I didn't see associated code change to mdt_general_osd_path_find. I think things may have changed out from under me while my review was in progress.